mods/voidmulti.sc2mod/base.sc2data/gamedata/abildata.xml

```diff
@@ -606,13 +672,28 @@
     </CAbilMorph>
     <CAbilMorph id="MorphToLurker">
         <Flags index="IgnoreFacing" value="1"/>
+        <Flags index="WaitUntilStopped" value="0"/>
```

Comment:
Fixed an issue where Morph to Lurker would be canceled with a Smart command issued immediately after the Morph command

The Lurker is not required to deaccelerate to speed 0 before the morph starts. During deacceleration, other commands could cancel this morph as it was still in its preparation phase.